### DISCUSSION TITLE
Max size of a workflow template YAML

### DISCUSSION THREAD

**Discussion by ryancurrah** [argoproj/argo-workflows#10444] [Q&A] [answered]
I'm trying to document the maximum size of a workflow yaml file. Does anyone happen to know what it is? 

Is it the max size of a kubernetes object which is 3MB?

https://stackoverflow.com/questions/60468110/kubernetes-object-size-limitations

**Comment by ljyanesm** [comment, marked as answer]
https://argo-workflows.readthedocs.io/en/latest/offloading-large-workflows/#:~:text=Argo%20stores%20workflows,an%20SQL%20database

There's some information there, but it is not abundantly clear what the limit is exactly given the caveats mentioned in the paragraph.
